### PR TITLE
Typos, references to `main.rs` supposed to be to `lib.rs`, and readability changes

### PR DESCRIPTION
**Pull request by dis-da-moe**
- Fixed the typos and grammatical issues I could find, most are in the tutorial docs but one is in code
- Fixed some cases where the tutorial said to change or add code in `main.rs` when it should be `lib.rs`
- Added alt attribute to an image

### CODE REVIEW

**Comment by HeavyRain266** [comment]
Why did you completely removed indentions between curly braces in some places?

**Comment by dis-da-moe** [comment]
> Why did you completely removed indentions between curly braces in some places?
Sorry that was unintentional I've pushed a commit that reverts that now.

**Comment by teoxoy** [comment]
Could you fixup this as well?
https://github.com/sotrh/learn-wgpu/commit/4d33b048c219c7eb1136b13ef81d8a44b497ab5e#r71315264

I wanted to open a PR but noticed this one was already in the works.

**Comment by dis-da-moe** [comment]
> Could you fixup this as well?

I was originally going to only do the tutorials I've read but I might as well go over the rest, I'll make a push soon for that.

File: docs/showcase/pong/README.md [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
 
-Practically the "Hello World!" of games. Pong has been remade thousands of times. I know Pong. You know Pong. We all know Pong. That being said, this time I wanted to put a little more effort than most people do. This showcase has a basic menu system, sounds, and different game states.
+Practically the "Hello World!" of games. Pong has been remade thousands of times. I know Pong. You know Pong. We all know Pong. That being said, this time I wanted to put in a little more effort than most people do. This showcase has a basic menu system, sounds, and different game states.
 
 The architecture is not the best as I prescribed to the "get things done" mentality. If I were to redo this project, I'd change a lot of things. Regardless, let's get into the postmortem.
 
 ## The Architecture
 
-I was messing around with separating state from the render code. It ended up similar to an entity component system.
+I was messing around with separating state from the render code. It ended up similar to an entity-component system.
```
Comment (softmoth) [inline docs/showcase/pong/README.md:11]: This is referring to an ECS. Use of that term is all over the map, but I think `Entity Component System model` would be the best way to refer to it here. "System" is a first-class element of the pattern here.

**Review by softmoth — CHANGES_REQUESTED** [review]
This is fantastic. I hope it gets merged!

I've found a few minor items that I think might be improved.

**Comment by dis-da-moe** [comment]
Thanks for the suggestions I've added those now

**Review by sotrh — APPROVED** [review]

**Pull request merged**